assets/addon_browser/addons.conf

```diff
@@ -507,5 +507,14 @@
     links: {
       "modrinth": "https://modrinth.com/mod/banners4bm"
     }
+  {
```

Comment:
Please add another `}` above this one.
The amount of brackets is currently not properly in balance.

Reply:
```suggestion
  }
  {
```